Makefile

```diff
@@ -69,7 +69,7 @@ compile:
 ################################################################################
 .PHONY: test
 test:
-	go test -v -timeout ${TEST_TIMEOUT} -mod=vendor ./...
```

Comment:
let's make it optional at least. it helps with debugging failures on prow. we can make it dependant on an env var - on prow we already set `CI=prow`

Reply:
oh, I thought we added that mostly by mistake.. I'll remove it from this PR, so we can address it properly

Reply:
Thanks